packages/gguf/src/gguf.ts

```diff
@@ -234,9 +271,10 @@ export async function gguf(
 		fetch?: typeof fetch;
 		additionalFetchHeaders?: Record<string, string>;
 		computeParametersCount?: boolean;
+		localFile?: boolean;
 	}
 ): Promise<GGUFParseOutput & { parameterCount?: number }> {
-	const r = new RangeView(url, params);
+	const r = params?.localFile ? new RangeViewLocalFile(url, params) : new RangeView(url, params);
```

Comment:
maybe we should rename the `url` variable here to `uri` since it can refer to local file

Reply:
Resolved with https://github.com/huggingface/huggingface.js/pull/656/commits/e68fc5075acd75da9a7fb0e143e63faee9f82ff7